IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m

```diff
@@ -681,7 +835,26 @@ - (NSString *)keychainGroupLoggingName
     {
         return s_defaultKeychainLabel;
     }
-
+    
     return _PII_NULLIFY(self.keychainGroup);
 }
+
++ (NSString *)teamId
+{
+    NSString* teamIdentifier = nil;
```

Comment:
nit: please create an util that only queries this team ID once like we did for iOS

Reply:
Done